hwilib/devices/trezor.py

```diff
@@ -258,7 +258,9 @@ def ignore_input():
                     # Add to txinputtype
                     txinputtype.multisig = multisig
                     if not psbt_in.witness_utxo:
```

Comment:
A better fix would be to actually change this line to be `if not is_wit`. Really what this is checking for is whether it is not segwit. In the segwit case, the type will already be `SPENDWITNESS` from line 234.